docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -42,8 +42,12 @@ To use Iceberg, you need:
   :ref:`AWS Glue catalog<iceberg-glue-catalog>`, a :ref:`JDBC catalog
   <iceberg-jdbc-catalog>`, a :ref:`REST catalog<iceberg-rest-catalog>`, or a
   :ref:`Nessie server<iceberg-nessie-catalog>`.
-* Network access from the Trino coordinator to the HMS. Hive metastore access
-  with the Thrift protocol defaults to using port 9083.
+* Data files must be in a
+  :doc:`supported file format <./object-storage-file-formats>`. These include:
+  
+  - ORC
+  - Parquet
+  - Avro
```

Comment:
Not sure but I think Avro is only for metadata and thats an implementation detail and should be left out here